common/events.h

```diff
@@ -91,7 +91,10 @@ enum EventType {
 	,
 	EVENT_JOYAXIS_MOTION = 23,
 	EVENT_JOYBUTTON_DOWN = 24,
-	EVENT_JOYBUTTON_UP = 25
+	EVENT_JOYBUTTON_UP = 25,
+	EVENT_CONTROLLERAXIS_MOTION = 26,
```

Comment:
Joystick and Controller events being something different is a quirk of SDL2. It would be much better if ResidualVM did not inherit the same flaw. Can we have an enum in ResidualVM defining what the joystick buttons are (based on the SDL game controller buttons) and in `ResVmSdlEventSource` something that tries to convert the old joystick button numbers into that enum ?

Reply:
The objective was to let the engine choose what to do, but I agree with you, it must be simplified.
Ok, I will try something like that.